docs/FAQ/gui-faq-en.md

```diff
@@ -20,147 +20,155 @@ Zonemaster
-shows something that you consider incorrect.
+#### 5. How can Zonemaster distinguish between what is right and wrong?
+There is not a definitive judgement of the health of a domain that can be bestowed by anyone.
+The people behind Zonemaster do not claim that the tool is correct in every aspect.
+Sometimes opinions differ.
+We are doing our very best to create a default policy for errors found within this project,
+with information primarly based on industry standards such as [RFCs].
+
+Hopefully you will find in our tool a good compromise between what is an actual,
+potentially dangerous *error* and what could be merely perceived as a *notice* or *warning*.
+You can find more information on our vision of severity level in the [Severity Level Definitions] document.
+However, note that one advantage of the tool is that it is always possible for one to add a policy file 
+better suited to one's needs and ask the tool to use that policy file when running tests.
+Such configuration is accessible through the "Options" button.
+
+Finally, as with all things as evolving as the DNS, what is a *notice* today could be an *error* tomorrow.
+If you think we have made a mistake in our judgement please do not hesitate to send us
+an email at [zonemaster-users@lists.iis.se] (moderated mailing list) with a link to your test
+result and an explanation as to why you think it shows something that you consider incorrect.
 
 <a name="q6"></a>
-#### 6. Does Zonemaster handle IPv6?
-Yes, it does. All tests run over IPv4 will also be run over IPv6 if Zonemaster
-is configured to do so.
+#### 6. Does Zonemaster support IPv6?
+Yes. By default Zonemaster will query name servers both over IPv4 and IPv6, unless explicitly
+configured otherwise.
+Such configuration is accessible through the "Options" button.
 
 <a name="q7"></a>
-#### 7. Does Zonemaster handle DNSSEC?
-Yes, if DNSSEC is available for a domain that is tested by Zonemaster, it will be
+#### 7. Does Zonemaster support DNSSEC?
+Yes. If DNSSEC is available for a domain that is tested by Zonemaster, it will be
 checked automatically.
+Note that you can also explicitly provide Delegation Signers (DS) records using the
+"Options" button.
 
 <a name="q8"></a>
 #### 8. What makes Zonemaster differ from other DNS zone validating software?
 Firstly, Zonemaster saves all history from earlier tests based on the tested
-domain, which means you can go back to a test you did a week ago and compare it
-to the test you ran a moment ago.
+domain, which means you can go back to a test you did some time ago and compare it
+to the test you ran just a moment ago.
 
-All tests that Zonemaster runs are defined in test case specifications that
-are linked from [Test Requirements
-document](https://github.com/zonemaster/zonemaster/blob/master/docs/requirements/TestRequirements.md)
+Secondly, all tests that Zonemaster runs are defined in Test Case specifications that
+can be found in the [Test Requirements] document.
```

Comment:
I think https://github.com/zonemaster/zonemaster/tree/master/docs/specifications/tests#list-of-defined-test-cases is a better link.

Reply:
Updated